packages/TypeDeclaration/tests/Rector/FunctionLike/ParamTypeDeclarationRector/ParamTypeDeclarationRectorTest.php

```diff
@@ -16,6 +16,7 @@ public function test(): void
             __DIR__ . '/Fixture/variadic.php.inc',
             __DIR__ . '/Fixture/mixed.php.inc',
             __DIR__ . '/Fixture/resource.php.inc',
+            __DIR__ . '/Fixture/nullable_resource.php.inc',
```

Comment:
Could you name the file `skip_*` → `skip_nullable_resource.php.inc`?

It's more clear from the name, that nothing should be changed

Reply:
Sure